llvm/lib/ObjCopy/ELF/ELFObjcopy.cpp

```diff
@@ -657,10 +657,14 @@ static Error verifyNoteSection(StringRef Name, endianness Endianness,
     NameSizeValue = byteswap(NameSizeValue);
     DescSizeValue = byteswap(DescSizeValue);
   }
+
+  // NameSizeValue and DescSizeValue need to be padded to 4.
+  auto iceil4 = [](auto v) { return (v + 3) / 4 * 4; };
```

Comment:
Rather than inventing a new function, why not use the existing `AlignTo` function?

Reply:
Done!